src/test/java/org/junit/rules/TestRuleAdapterTest.java

```diff
@@ -0,0 +1,59 @@
+package org.junit.rules;
+
+import static java.util.Arrays.asList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.experimental.results.PrintableResult.testResult;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runners.model.FrameworkMethod;
+import org.junit.runners.model.Statement;
+
+public class TestRuleAdapterTest {
```

Comment:
Thia test needs to be added to the appropriate suite or it won't run as part of the Maven build